ios/MaasCore/Sources/MaasComponents/Supporting Files/CircularSpinner.swift

```diff
@@ -0,0 +1,88 @@
+struct Spinner: View {
```

Comment:
Do we need some custom KMP constants for Spinner ? 🤔

Reply:
Only if they will be used in Kotlin 🤔 
But probably it makes sense to use default spinner.
@justasm do you have default spinner with Jetpack compose?

Reply:
The Material implementation in Compose UI has a `CircularProgressIndicator` that looks pretty much the same and allows customizing the `color: Color` and `strokeWidth: Dp`.

Reply:
ok then, if it's only strokeWidth, it's probably not worth defining as SpinnerConstants..